packages/components/src/components/post-menu/post-menu.tsx

```diff
@@ -71,20 +72,27 @@ export class PostMenu {
 
   componentDidLoad() {
     this.popoverRef.addEventListener('postToggle', (event: CustomEvent<boolean>) => {
-      this.isVisible = event.detail;
-      this.toggleMenu.emit(this.isVisible);
-  
-      requestAnimationFrame(() => {
-        if (this.isVisible) {
-          this.lastFocusedElement = this.root.activeElement as HTMLElement;
-          const menuItems = this.getSlottedItems();
-          if (menuItems.length > 0) {
-            (menuItems[0] as HTMLElement).focus();
-          }
-        } else if (this.lastFocusedElement) {
-          this.lastFocusedElement.focus();
+      eventGuard.call(
+        this,
+        event,
+        { targetLocalName: 'post-popovercontainer' },
```

Comment:
If the post-popovercontainer here always is a child of the post-menu, we should use the delegatorSelector with the value `post-menu`.